import_module/create_groups.py

```diff
@@ -205,11 +209,15 @@ def get_horizontal_slice(extent, geomcol, zoom):
 
         if sliced_poly:
             if sliced_poly.GetGeometryName() == 'POLYGON':
-                slice_collection.AddGeometry(sliced_poly)
+                slice_infos['tile_y_top'].append(TileY)
+                slice_infos['tile_y_bottom'].append(TileY+3)
+                slice_infos['slice_collection'].AddGeometry(sliced_poly)
                 #logging.info('sliced poly: %s' % sliced_poly)
             elif sliced_poly.GetGeometryName() == 'MULTIPOLYGON':
                 for geom_part in sliced_poly:
-                    slice_collection.AddGeometry(geom_part)
+                    slice_infos['tile_y_top'].append(TileY)
+                    slice_infos['tile_y_bottom'].append(TileY + 3)
+                    slice_infos['slice_collection'].AddGeometry(geom_part)
                     #logging.info('sliced poly: %s' % sliced_poly)
```

Comment:
Would these be helpful to debug? If so, maybe switch to logging.debug and uncomment?

Reply:
good question. I think, I never used this info in the last times. so maybe I will just remove it.